greeting-guestbook/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "greeting-emails",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "tsc",
+    "test": "npx knex migrate:rollback && npx knex migrate:up && jest",
+    "lint": "eslint src",
+    "lint-fix": "eslint --fix src"
+  },
+  "devDependencies": {
+    "@dbos-inc/dbos-cloud": "^1.15.9",
+    "@dbos-inc/eslint-plugin": "^3.1.0",
+    "@types/jest": "^29.5.5",
+    "@types/supertest": "^2.0.14",
+    "eslint": "^8.56.0",
+    "jest": "^29.7.0",
+    "supertest": "^7.0.0",
+    "ts-jest": "^29.1.1",
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "@dbos-inc/dbos-sdk": "^1.15.9",
```

Comment:
Use our latest SDK and Cloud CLI version `1.18.8`

Reply:
done